tests/test_osparc.py

```diff
@@ -132,13 +132,17 @@ def test_osparc_successful_simulation(client):
     }
     r = client.post("/start_simulation", json=data)
     assert r.status_code == 200
-    check_simulation_data = json.loads(r.data)["data"]
+    json_data: dict = json.loads(r.data)
```

Comment:
For consistency, I would remove the `: dict` (same for line 173) since it's not used on line 142 (line 180, resp.).

Reply:
Yep, makes sense. I have fixed this.

Reply:
Can I get you to merge @agarny? It seems I don't have the right permissions to do that 🤔

Reply:
Thanks @bisgaard-itis. It looks like I am not authorised either. @alan-wu, could you please check and merge this PR?